src/answers-umd.js

```diff
@@ -452,33 +427,57 @@ class Answers {
     });
   }
 
+  /**
+   * A promise that resolves when ponyfillCssVariables resolves,
+   * or resolves immediately if ponyfill is disabled
+   * @param {boolean} option to opt out of the css variables ponyfill
+   * @return {Promise} resolves after ponyfillCssVariables, or immediately if disabled
+   */
+  _handlePonyfillCssVariables (ponyfillDisabled) {
+    return new Promise((resolve, reject) => {
```

Comment:
nit but I think we can shorten this one slightly too, by doing
```js
if (!ponyfillDisabled) {
  return Promise.resolve();
}
return this.ponyfillCssVariables()
```
Another option, though, might be to have ponyfillCssVariables() go back to just being a pass through to cssVars(), and have all of the promise logic go into this _handle function? I think that divides up responsibility a little more neatly (let me know if you have any other thoughts). Then this _handle method could return a Promise.resolve() if ponyfills are disabled, or a new Promise to passes resolve into the onFinally of ponyfillCssVariables if not disabled

Reply:
I like that idea. It's definitely a little cleaner